pages/in/tx/iscn/_id/index.vue

```diff
@@ -396,6 +396,25 @@ export default {
     totalStakeholdersShares() {
       return this.stakeholders.reduce((t, s) => t + s.rewardProportion, 0);
     },
+    creatorLikerId() {
+      if (this.creatorId.startsWith(EXTERNAL_URL)) {
+        this.creatorId.replace(EXTERNAL_URL, '');
+      }
+      return '';
+    },
+    isCreatorIdURL() {
+      try {
+        const res = new URL(this.creatorId);
+        return !!res;
+      } catch (err) {
+        return false;
+      }
+    },
+    creatorLink() {
+      if (this.creatorLikerId) return this.getCreatorPortfolioURL(this.creatorLikerId);
```

Comment:
`this.creatorLikerId` always return `''` so this will never run?

Reply:
Should return a liker id after the fix above